### ISSUE TITLE
Cannot add distro

### ISSUE THREAD

**Issue by FranklinYu** [release-monitoring/anitya#791]
I want to add MacPorts. When I did that through [web interface](https://release-monitoring.org/distro/add) I got HTTP 500 on form submission.

**Comment by Zlopez** [comment]
Fixed in master #757

**Comment by FranklinYu** [comment]
I assume that it will be deployed only after a new release is created? Is there any schedule/plan for that?

**Comment by Zlopez** [comment]
I don't have any schedule right now, but it's probably a good time to do a release.
I just check what is merged right now and release a new version, probably next week.

**Comment by sroracle** [comment]
This issue appears to be happening again. I have the exact same problem.

**Comment by Zlopez** [comment]
It looks like I fixed only half of the issue :-(

I fixed the issue when you open packages in one distribution, but I missed this one. Reopening.

**Comment by sroracle** [comment]
For the record, this still appears to be happening, but it's because the flagship instance is still on 0.16.1.

**Comment by Zlopez** [comment]
@sroracle Sorry for this, I found out some new issue on staging and need to fix it.

If the fix will work, I will deploy a new version on production soon.

**Comment by z3ntu** [comment]
It would be really great if important fixes for issues like this one, would be rolled out faster and not just after months.

**Comment by Zlopez** [comment]
I fixed last one issue on new version of Anitya few days ago and I want to deploy a new version tomorrow.

I'm really sorry for the delay.

**Comment by FranklinYu** [comment]
Fix verified! Thanks.
